Opencast-Moodle/moodle-block_opencast
# Avoid escaping plus sign upon updating metadata

This PR fixes #291 

### Description
the plus sign is interpreted as whitespace when updating the metadata for both events and series.

### Solution
By walking through the array of metadata and using `urlencode` for both title and description, the problem is resolved.

### How to Test this PR
Before patching this PR, try to change the title and description metadata values to add "+" to those fields under the updatemetadata function:
Overview Page > Current Videos Table > Action column > update metadata
You will notice that the result shown no '+' but a whitespace.

After patching this PR, repeat the previous process and the '+' must be shown.
